frontend/src/api/students.ts

```diff
@@ -0,0 +1,64 @@
+import { POST, handleAPIError } from "../api/requests";
+import { Contact, StudentData } from "../components/StudentForm/types";
+
+import type { APIResult } from "../api/requests";
+
+export type Student = {
+  _id: string;
+  student: Contact;
+  emergency: Contact;
+  serviceCoordinator: Contact;
+  location: string;
+  medication?: string;
+  birthday: Date;
+  intakeDate: Date;
+  tourDate: Date;
+  regular_programs: string[];
+  varying_programs: string[];
+  dietary: string[];
+  otherString?: string;
+};
+
+type StudentJSON = {
+  _id: string;
+  student: Contact;
+  emergency: Contact;
+  serviceCoordinator: Contact;
+  location: string;
+  medication?: string;
+  birthday: string;
+  intakeDate: string;
+  tourDate: string;
+  regular_programs: string[];
+  varying_programs: string[];
+  dietary: string[];
+  otherString?: string;
+};
+
+function parseStudent(studentJSON: StudentJSON): Student {
+  return {
+    _id: studentJSON._id,
+    student: studentJSON.student,
+    emergency: studentJSON.emergency,
+    serviceCoordinator: studentJSON.serviceCoordinator,
+    location: studentJSON.location,
+    medication: studentJSON.medication,
+    birthday: new Date(studentJSON.birthday),
+    intakeDate: new Date(studentJSON.intakeDate),
+    tourDate: new Date(studentJSON.tourDate),
+    regular_programs: studentJSON.regular_programs,
+    varying_programs: studentJSON.varying_programs,
+    dietary: studentJSON.dietary,
+    otherString: studentJSON.otherString,
+  } as Student;
+}
+
+export async function createStudent(student: StudentData): Promise<APIResult<Student>> {
```

Comment:
It may be good to add a check here to make sure that the contact information (email, phone) provided of the emergency contact is different from the contact information (email, phone) of the student